### PR TITLE
[MM-16389] Reference icon_emoji parameter for webhooks

### PR DESCRIPTION
**Pull request by krjn**
An icon_emoji parameter was added to incoming webhook requests, that overrides the profile icon and the icon_url parameter.

https://mattermost.atlassian.net/browse/MM-16389

Related PRs:
https://github.com/mattermost/mattermost-server/pull/11586
https://github.com/mattermost/mattermost-webapp/pull/3074
https://github.com/mattermost/mattermost-mobile/pull/2961
https://github.com/mattermost/mattermost-redux/pull/876
https://github.com/mattermost/docs/pull/2832

### CODE REVIEW

File: site/content/integrate/incoming-webhooks/_index.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Incoming webhooks support more than just the `text` field. Here is a full list o
...
 | username | Overrides the username the message posts as.<br> Defaults to the username set during webhook creation or the webhook creator's username if the former was not set.<br> Must be enabled [in the configuration](https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames). | No |
 | icon\_url | Overrides the profile picture the message posts with.<br> Defaults to the URL set during webhook creation or the webhook creator's profile picture if the former was not set.<br> Must be enabled [in the configuration](https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons). | No |
+| icon\_emoji | Overrides the profile picture and `icon_url` parameter. Defaults to none and is not set during webhook creation. Must be enabled [in the configuration](https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons). | No |
```
Comment (lieut-data) [inline site/content/integrate/incoming-webhooks/_index.md]: Wondering if we can explain in more detail what to provide for this parameter? e.g. do I write the emoji as `:tada:` or `tada`?

File: site/content/integrate/incoming-webhooks/_index.md [hunk 2]
```diff
@@ -84,6 +85,8 @@ Content-Length: 630
 }
 ```
 
+Optionally, `icon_url` could be replaced by `icon_emoji: "trophy"`.
```
Comment (lieut-data) [inline site/content/integrate/incoming-webhooks/_index.md]: nit: this comment seems out of place -- maybe just remove?

**Comment by hmhealey** [comment]
Merging this now since the feature got into 5.14

**Comment by hmhealey** [comment]
Thanks again, @krjn!

**Pull request merged**